tests/v2/validation/deleting/delete_init_machine_test.go

```diff
@@ -48,21 +48,21 @@ func (d *DeleteInitMachineTestSuite) TestDeleteInitMachine() {
 	err = v1.ConvertToK8sType(cluster, &updatedCluster)
 	require.NoError(d.T(), err)
 
-	name := "K3S"
+	var name string
 
 	if strings.Contains(updatedCluster.Spec.KubernetesVersion, "rke2") {
 		name = "RKE2"
+	} else {
```

Comment:
imo the original way was more concise

Reply:
I don't have a preference, I can put it back. Just following the practice in the other files.

Reply:
Resolved.